mediainfo-gui.rb

```diff
@@ -32,4 +32,8 @@ def install
       system "make", "install"
     end
   end
+
+  test do
+    assert_predicate "#{bin}/mediaino-gui", :exist?
```

Comment:
Why? A test is now mandatory?

Reply:
It is good practice in Homebrew to have a test… which should do “something” useful… what is not always possible. (When I cannot include a “real” test, I often just call the help. That way all Homebrew tests do pass.)

Reply:
@JeromeMartinez I added formula validation step when we migrated to github-actions.

mediainfo-gui pop the GUI even whith simple --version or --help argument, so I can't use it.